doc/controls/ShadowContainer.md

```diff
@@ -0,0 +1,288 @@
+---
+uid: Toolkit.Controls.ShadowContainer
+---
+
+## Summary
+The `ShadowContainer` provides the possibility to add many-colored shadows to its content.
+
+## Remarks
+For now, the control simply adapts its corner radius to the content's corner radius. More complicated shapes, such as text or pictures with alpha, are not supported.
+
+### XAML
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<utu:ShadowContainer>
+    <utu:ShadowContainer.Shadows>
+        <utu:ShadowCollection>
+            <utu:Shadow ... />
+            <utu:Shadow ... />
+        </utu:ShadowCollection>
+    </utu:ShadowContainer.Shadows>
+
+    <SomeControl />
+
+</utu:ShadowContainer>
+```
+
+### Inheritance
+Object &#8594; DependencyObject &#8594; UIElement &#8594; FrameworkElement &#8594; Control &#8594; ContentControl > ShadowContainer
+
+### Properties
+| Property | Type             | Description |
+| -------- | ---------------- | ----------- |
+Shadows  | ShadowCollection |  The collection of shadows that will be displayed under your control. A `ShadowCollection` can be stored in a resource dictionary to have a consistent style throughout your app. The `ShadowCollection` implements `INotifyCollectionChanged`.
+
+## Shadow
+
+Dependency object representing a single shadow.
+
+### Shadow Properties
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+IsInner | bool | True if this shadow is an inner shadow (similar to `inset` of `box-shadow` in CSS).
+OffsetX | double | The X offset of the shadow.
+OffsetY | double | The Y offset of the shadow.
+Color | Color | The color of the shadow. It will be multiplied by the `Opacity` property before rendering.
+Opacity | double | The opacity of the shadow.
+BlurRadius | double | The radius of the blur that will be applied to the shadow **[0..100]**.
+Spread | double | The spread will inflate or deflate (if negative) the control shadow size **before** applying the blur.
+
+## Usage
+
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<Page.Resources>
+	<Color x:Key="UnoColor">#7a67f8</Color>
+	<Color x:Key="UnoPink">#f85977</Color>
+
+	<!-- You can define your shadows in the resource dictionary -->
+	<ui:ShadowCollection x:Name="ButtonShadows">
```

Comment:
Shouldn't this be a `x:"Key"`?
```suggestion
	<ui:ShadowCollection x:Key="ButtonShadows">
```

Reply:
preferably yes,
but doesnt necessarily need to be

> [x:Name](https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/x-name-attribute) can be used instead of [x:Key](https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/x-key-attribute). However, x:Name also generates a code behind field for the resource. As a result, x:Name is less efficient than x:Key because that field needs to be initialized when the page is loaded.
> -- https://learn.microsoft.com/en-us/windows/apps/design/style/xaml-resource-dictionary

Reply:
Good to know, thanks @Xiaoy312

Reply:
preferably yes but since this is a backport PR, an update for this would be on main branch first